client/src/components/Community/Community.css

```diff
@@ -20,12 +20,15 @@
 }
 
 .community-component .title {
-  opacity: 0;
-  width: 0;
+  opacity: 1;
+  width: 20em;
   height: 0;
   pointer-events: none;
-  margin: 0;
+  margin: 3em;
```

Comment:
try changing the margin property to `margin-bottom` ,as currently a lot of margin is there on the top of the heading

Reply:
All right, I have fixed it.